src/components/Canvas.js

```diff
@@ -1,27 +1,235 @@
-import React, { useRef, useEffect } from "react";
+import React, { useRef, useEffect, useState } from "react";
+import * as faceapi from "face-api.js";
+import { connect } from "react-redux";
+import {
+	setStudentLoginStatus,
+	setStudentNewStatus,
+	setStudentFirstName,
+	setStudentLastName,
+	setStudentId,
+	addNewStudent,
+} from "../redux/actions/student.js";
+import { fetchStudentInfo } from "../server.js";
 import Panel from "./Panel.js";
 import "./Canvas.css";
 
-const Canvas = () => {
+const MODELS_URI = `${process.env.PUBLIC_URL}/models`;
+const LABELS_URI = `${process.env.PUBLIC_URL}/labels`;
+const NUM_IMGS_PER_LABEL = 10;
```

Comment:
can't access the local file system in a web environment so the number of images read is hardcoded